src/document/doc-printer.js

```diff
@@ -154,9 +154,7 @@ function fits(next, restCommands, width, options, mustBeFlat) {
     }
 
     const x = cmds.pop();
-    const ind = x[0];
-    const mode = x[1];
-    const doc = x[2];
+    const [ind, mode, doc] = x;
```

Comment:
```suggestion
    const [ind, mode, doc] = cmds.pop();
```

Reply:
I can't see the variable `x` serving it's purpose anywhere else.

Reply:
Good idea, updated. Thanks!